### DISCUSSION TITLE
Free Software licence?

### DISCUSSION THREAD

**Discussion by jeannekamikaze** [snakers4/silero-models#259] [Ideas]
Is there any chance the Silero models could be published under a license compatible with free software licenses? Looking at the [license history](https://github.com/snakers4/silero-models/commits/master/LICENSE), it appears that the project was originally under the AGPL, but was recently re-licensed under CC BY-NC-SA. The latter does not appear to be [compatible](https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses/) with free software licenses due to the restriction on commercial use.

Thank you.

**Comment by snakers4** [comment]
>  due to the restriction on commercial use.
> Thank you.

Hi,
This is by design.

**Comment by jeannekamikaze** [comment]
And is that Silero's design, or a carry-over from one of the models that Silero TTS uses internally?
